packages/shared/components/molecules/NftActivityDetails.svelte

```diff
@@ -19,16 +21,36 @@
 
     $: nft = getNftByIdFromAllAccountNfts($selectedAccountIndex, activity.nftId)
     $: isTimelocked = activity?.asyncData?.timelockDate > $time
+
+    function handleClick(): void {
+        $selectedNftId = activity.nftId
+        $dashboardRouter.goTo(DashboardRoute.Collectibles)
+        $collectiblesRouter.goTo(CollectiblesRoute.Details)
+        closePopup()
+    }
 </script>
 
 <nft-transaction-details class="w-full space-y-6 flex flex-auto flex-col flex-shrink-0">
     <main-content class="flex flex-auto w-full flex-col items-center justify-center space-y-3 overflow-hidden">
-        <nft-summary class="flex w-full items-center justify-center w-full space-x-2">
-            <NftImageOrIconBox nftId={activity.nftId} size="small" />
-            <Text type={TextType.h3} fontWeight={FontWeight.semibold} classes="whitespace-pre truncate">
-                {nft?.name}
-            </Text>
-        </nft-summary>
+        {#if $selectedAccountNfts.some((nft) => nft.id === activity.nftId)}
```

Comment:
you could just disable the `button` instead of copying the component

Reply:
I was still getting the button cursor when hovering over it and wasn't sure the best way to disable it. Guess I could give it a class that toggles whether the button is active.

Reply:
Went ahead isolated it all to a single button and added a conditional class to change the cursor styling